### ISSUE TITLE
Multiple spinners when synchronizing

### ISSUE THREAD

**Issue by janvlug** [alainm23/planify#1460]
**Describe the bug**
Sometimes I see two (maybe sometimes even more, not sure) spinners when Planify is updating against my Nextcloud back-end:

![image](https://github.com/user-attachments/assets/8029346a-acdd-4b62-b342-37ded5b99c65)

Is this normal? I experience a few other issues with synchronization: https://github.com/alainm23/planify/issues/1391, https://github.com/alainm23/planify/issues/1457. Could these be related?

**To Reproduce**
1. Open Planify that syncs against a Nextcloud back-end.
2. See how synchronization starts

**Expected behavior**
It should be clear for me if synchronizing projects (task lists) goes in parallel or serial. Currently, it looks sometimes in parallel, but as a user I do not know if this is expected or not.

**Desktop (please complete the following information):**
 - Fedora, GNOME, Wayland
 - Planify version: 4.11.4

**Comment by janvlug** [comment]
I noticed now also that the spinner (in general I see just one) does not always move linearly from the top to the bottom. For a few sub-tasks it jumps up and down in the list.

**Comment by janvlug** [comment]
I made a screencast of the non-linearity:

https://github.com/user-attachments/assets/b6142389-f0f3-4680-a88d-2343a0352bec

**Comment by janvlug** [comment]
Also collapsing or not a Task list with sub-tasks seems to influence the order of synchronizing (e.g. collapsing the "Dagtaken" in the screen-cast above seems to change the synchronization order.

**Comment by alainm23** [comment]
Unrelated, spinners were added to indicate the synchronization process of each project.
